src/request.js

```diff
@@ -225,6 +225,23 @@ class Request {
       this.client.instance().numberInsight.get({level: 'standard', number: number}, this.response.insightStandard.bind(this.response));
     });
   }
+
+  // sending messages
+
+  sendSms(first, second, third, command) {
```

Comment:
When the number of arguments isn't set you normally see `arguments` used, so I found that a bit confusing.

In ES2015 there's `...` which is used for the "Rest of the arguments". See https://babeljs.io/docs/learn-es2015/#default--rest--spread

The parameter names - to me - are a little unclear right now. But this is just a nitpick.